pkg/verifier/notation/notation.go

```diff
@@ -187,19 +207,35 @@ func (v *notationPluginVerifier) verifySignature(ctx context.Context, subjectRef
 	return (*v.notationVerifier).Verify(ctx, subjectDesc, refBlob, opts)
 }
 
+func isValidTrustStoreType(storeType string) bool {
+	return slices.Contains(trustStoreTypes, storeType)
+}
+
 func parseVerifierConfig(verifierConfig config.VerifierConfig, namespace string) (*NotationPluginVerifierConfig, error) {
 	verifierName := verifierConfig[types.Name].(string)
 	conf := &NotationPluginVerifierConfig{}
-
 	verifierConfigBytes, err := json.Marshal(verifierConfig)
 	if err != nil {
 		return nil, re.ErrorCodeConfigInvalid.NewError(re.Verifier, verifierName, re.EmptyLink, err, nil, re.HideStackTrace)
 	}
-
 	if err := json.Unmarshal(verifierConfigBytes, &conf); err != nil {
 		return nil, re.ErrorCodeConfigInvalid.NewError(re.Verifier, verifierName, re.EmptyLink, err, fmt.Sprintf("failed to unmarshal to notationPluginVerifierConfig from: %+v.", verifierConfig), re.HideStackTrace)
 	}
 
+	// convert <store>:<certs> to ca:<store><certs>
+	// TBD if have both legacy and new types of store struct, then throw err
```

Comment:
Try to avoid comments like `TBD` or `TODO`. You may add a comment in this PR like this one. And then create a new issue to track and put link in the comment.